home/src/main/resources/rdf/i18n/en_US/interface-i18n/firsttime/vitro_UiLabel.ttl

```diff
@@ -6955,6 +6955,14 @@ uil-data:upload_new_css.Vitro
         uil:hasKey       "upload_new_css" ;
         uil:hasPackage   "Vitro-languages" .
 
+uil-data:cusotm_css_style.Vitro
+        rdf:type         owl:NamedIndividual ;
+        rdf:type         uil:UILabel ;
+        rdfs:label       "Custom css file"@en-US ;
+        uil:hasApp       "Vitro" ;
+        uil:hasKey       "cusotm_css_style" ;
```

Comment:
@milospp could you just correct the typo on the key name ("custom_css_style" instead of cusotm_css_style") ?